app/backend/models/user-follow.js

```diff
@@ -0,0 +1,42 @@
+const {mongoose} =  require('../db');  // The mongodb connector library
+
+let UserFollow = mongoose.model('UserFollow', {
```

Comment:
Fields for names and surnames seem redundant, as those can be retrieved via user's ID.